### PR TITLE
examples/nxscope: add support for CDCACM dev

### PR DESCRIPTION
**Pull request by raiden00pl**
## Summary

- logging/nxscope/nxscope_iser.c: wait for device if CONFIG_SERIAL_REMOVALBE=y
- examples/nxscope: add support for CDCACM dev

## Testing
stm32f4discovery

### CODE REVIEW

File: logging/nxscope/nxscope_iser.c [hunk 1]
```diff
@@ -177,13 +177,35 @@ int nxscope_ser_init(FAR struct nxscope_intf_s *intf,
...
+      /* Try to open the console */
+
+      priv->fd = open(priv->cfg->path, flags);
+      if (priv->fd < 0)
+        {
+          /* ENOTCONN means that the USB device is not yet connected.
+           * Anything else is bad.
+           */
+
+          DEBUGASSERT(errno == ENOTCONN);
```
Comment (xiaoxiang781216) [inline logging/nxscope/nxscope_iser.c]: should we return fail for other errno

File: logging/nxscope/nxscope_iser.c [hunk 2]
```diff
@@ -189,7 +189,10 @@ int nxscope_ser_init(FAR struct nxscope_intf_s *intf,
            * Anything else is bad.
            */
 
-          DEBUGASSERT(errno == ENOTCONN);
+          if (errno != ENOTCONN)
```
Comment (xiaoxiang781216) [inline logging/nxscope/nxscope_iser.c:192]: move to the first patch

File: logging/nxscope/nxscope_iser.c [hunk 3]
```diff
@@ -199,13 +202,13 @@ int nxscope_ser_init(FAR struct nxscope_intf_s *intf,
   while (priv->fd < 0);
 #else
   priv->fd = open(priv->cfg->path, flags);
+#endif
   if (priv->fd < 0)
     {
-      _err("ERROR: failed to open %s\n", priv->cfg->path);
+      _err("ERROR: failed to open %s %d\n", priv->cfg->path, errno);
```
Comment (xiaoxiang781216) [inline logging/nxscope/nxscope_iser.c:208]: to the first patch too

Comment (raiden00pl) [reply logging/nxscope/nxscope_iser.c:208]: Ouch, fixup to wrong commit. Fixed

**Pull request merged**